docs/projects/coordinating-multiple-projects/deploy-release-step/index.md

```diff
@@ -13,7 +13,19 @@ when you add a _Deploy Release_ step to your deployment process, you can then se
 
 You can add many _Deploy Release_ steps to your process, if you wish to deploy releases of many projects.
 
-When creating a release of a project containing _Deploy Release_ steps you can select the release version of each project, similar to the way versions of packages are selected.  
+## Creating a release
+
+When creating a release of a project containing _Deploy Release_ steps you can select the release version of each project, similar to the way versions of packages are selected:
```

Comment:
Minor thing, but the step name is _Deploy a Release_

Don't know if it matters though. I expect most people would know that's what we mean?

Reply:
Updated, I think I prefer the wording as _Deploy a release_